### DISCUSSION TITLE
Code Push 도입

### DISCUSSION THREAD

**Discussion by CaffeinCoding** [KNUT-Capstone-Design-team-1/wip-application-v2#3] [Ideas]
### V2 앱에 Code Push 도입이 괜찮을까요
현재 앱의 규모 상으로는 Code Push가 굳이 필요 없을 것 같기도 한데, 초기에 도입하는 것도 나쁘지 않아 보여서 의견을 제시합니다

**Comment by YongSiHun** [comment]
네이티브 앱에서도 유효한가요??
웹 앱에서 더 효과적일 것 같아서 웹 앱으로 결정되면 하면 좋을 것 같습니다

**Reply by CaffeinCoding** [reply]
네이티브 앱 또는 네이티브 + 웹앱으로 할 때를 기준으로 의견을 제시했습니다
웹앱이면 어차피 웹서비스라서 code push는 굳이 필요가 없을 것 같습니다
